sass/_QuerySuggestPreview.scss

```diff
@@ -18,20 +18,19 @@
 }
 
 .coveo-suggestion-container {
-  display: inline-flex;
-  flex-wrap: nowrap;
-  align-items: stretch;
-  min-width: 100%;
+  width: 100%;
+  display: flex;
 
   .coveo-magicbox-suggestions {
     border: none;
     float: left;
+    flex-basis: 30%;
```

Comment:
Could use a variable here to make clear/ensure that this line and line 33 is always 100%.

Reply:
Good idea, I'll add a variable.